sopel/plugins/capabilities.py

```diff
@@ -25,7 +36,29 @@
 
 
 class Manager:
-    """Manager of Capability Requests."""
+    """Manager of plugin's capability requests.
```

Comment:
```suggestion
    """Manager of plugins' capability requests.
```

The manager manages _all_ plugins' requests, and not one manager instance for each plugin, right?

(Can also just be "plugin capability requests" with no possessive at all.)

Reply:
I'm honestly lost with the usage of possessive or lack of possessive to express slightly different things. The nuances are too subtle for me to detect and I can't get around.